builtin/providers/aws/resource_aws_api_gateway_domain_name_test.go

```diff
@@ -205,10 +207,28 @@ func testAccAWSAPIGatewayDomainNameConfigCreate(name string) string {
 	return fmt.Sprintf(`
 resource "aws_api_gateway_domain_name" "test" {
   domain_name = "%s"
-  certificate_body = "%v"
-  certificate_chain = "%v"
+  certificate_body = <<EOF
+%vEOF
```

Comment:
This is done on purpose to avoid a diff issue. The other solution would be to update line 43 for instance, adding:

```go
...
resource.TestCheckResourceAttr("aws_api_gateway_domain_name.test", "certificate_body", fmt.Sprintf("%v\n", testAccAWSAPIGatewayCertBody),
...
```